krun/scheduler.py

```diff
@@ -112,7 +125,7 @@ def _parse(self):
             assert False
 
         # Check we saw all the necessary header fields
-        assert seen_headers == ManifestManager.HEADER_FIELDS
+        assert ManifestManager.HEADER_FIELDS.issubset(seen_headers)
```

Comment:
I'm not sure I understand this, it seems to say that the header fields in the manifest should be a subset of the headers that have been read so far by the parser. Is that right?